### PR TITLE
Notebook2 header / style prep 

### PR DESCRIPTION
**Pull request by akotlar**
This simply adds the wrapping structure + header from app.hail.is.

Includes no scss from app.hail.is. That will be the next PR unless you want it here. We should also decide whether we want a home page (the smoothly-rising Hail that Arcturus liked), i.e whether this will be extended with a link to batch, or not (we discussed the option at our last meeting)

cc @cseed

### CODE REVIEW

File: notebook2/notebook/templates/index.html [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-            <option value="{{ image }}"{{'selected' if image == default else ''}}>{{ image }}</option>
-            {% endfor %}
-          </select>
-        </p>
-        <input type="submit" value="Start a Hail Notebook" />
-      </form>
-    </div>
+  <body id='theme-site'>
+    <div>{% include 'header.html' %}</div>
+    <span id="main">
```
Comment (danking) [inline notebook2/notebook/templates/index.html]: It seems weird to use a `span` for something that's not displayed inline, even if the subsequent styling overrides the display to `flex`. Thoughts on using a `div` instead?

File: notebook2/notebook/templates/header.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<span id="header">
```
Comment (danking) [inline notebook2/notebook/templates/header.html]: same argument here about span vs div.

File: notebook2/notebook/templates/index.html (hunk 1 above)
Comment (akotlar) [reply notebook2/notebook/templates/index.html]: Yeah, it may be more semantically correct to use div.

**Comment by akotlar** [comment]
@danking thanks, done

**Pull request merged**